components/org.wso2.carbon.metrics.data.service/src/main/java/org/wso2/carbon/metrics/data/service/MetricsDataService.java

```diff
@@ -191,6 +185,37 @@ private long getStartTime(String from) {
         return sourcesList.toArray(new String[sourcesList.size()]);
     }
 
+    public MetricHierarchyData getHierarchy(String source, String path) {
```

Comment:
@msmshariq Fixed as of #39. commit:[84fd967](https://github.com/grainier/carbon-metrics/commit/84fd9673646d8c1bb97a06dce875ecc3e45722cf)